ramlfications/errors.py

```diff
@@ -49,7 +49,7 @@ class InvalidSecuritySchemeError(BaseRAMLError):
 # Loader Exceptions
 #####
 
-class LoadRAMLError(Exception):
+class LoadRAMLError(BaseRAMLError):
```

Comment:
So the original purpose of `BaseRAMLError` was for errors related to _parsing_ a RAML file. It was not meant to be used for anything outside of `ramlfications/parser.py`, or now `ramlfications/parser/*`. 

I can see a case for having a base exception class for the RAMLfications package, that a base RAML parser error inherits as well as a loader error, invalid version error, etc.